soc/arm/nxp_imx/rt/soc_rt10xx.c

```diff
@@ -200,14 +200,26 @@ static ALWAYS_INLINE void clock_init(void)
 
 #if DT_NODE_HAS_STATUS(DT_NODELABEL(enet), okay) && CONFIG_NET_L2_ETHERNET
 	/* Enable clock output for ENET1 */
-	IOMUXC_EnableMode(IOMUXC_GPR, kIOMUXC_GPR_ENET1TxClkOutputDir, true);
+#ifdef CONFIG_ETH_MCUX_RMII_EXT_CLK
+	IOMUXC_EnableMode(IOMUXC_GPR, kIOMUXC_GPR_ENET1RefClkMode, true);
+	IOMUXC_EnableMode(IOMUXC_GPR, kIOMUXC_GPR_ENET1TxClkOutputDir, false);
+#else
```

Comment:
My understanding, is that when using an external refclk. The RefClkMode needs to be true, and OutputDir needs to be false.

When the RT1064 is generating the refclk then, RefClkMode should be false, and the OutputDir should be true.

Reply:
I'm working on a custom board with RT1052.
To use an external clock for Ethernet, I only have to configure `kIOMUXC_GPR_ENET1TxClkOutputDir` as input (see  #59841)

Without that, Ethernet is extremely slow (about few kbits/s).